src/game/client/system/particlesystem/particle.cpp

```diff
@@ -185,7 +187,9 @@ void Particle::Compute_Color_Rate()
  */
 void Particle::Apply_Force(const Coord3D &force)
 {
-    m_accel += force;
+    m_accel.x += force.x;
+    m_accel.y += force.y;
+    m_accel.z += force.z;
```

Comment:
I agree with Duncan on the operators. This specific aspect does not look like a worthwhile change, because 
```
    m_accel.x += force.x;
    m_accel.y += force.y;
    m_accel.z += force.z;
```
is a lot more to read than
```
   m_accel += force;
```
Code should be simple.

Reply:
I added operators based on what Vector2 and Vector3 have and used them where appropriate.